src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1081,32 +1081,32 @@ void EnGo_DrawRolling(EnGo* this, PlayState* play) {
 
 s32 EnGo_OverrideLimbDraw(PlayState* play, s32 limb, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
     EnGo* this = (EnGo*)thisx;
-    Vec3s vec1;
-    f32 float1;
+    Vec3s limbRot;
+    f32 temp;
```

Comment:
this temp is fake, you can remove it and "inline" its uses instead

e.g.
```c
temp = Math_SinS(this->jointTable[limb]);
rot->y += temp * 200.0f;
```
->
```c
rot->y += Math_SinS(this->jointTable[limb]) * 200.0f;
```